scripts/reset-test-vm.mjs

```diff
@@ -25,15 +25,15 @@ const runAnsiblePlaybooks = async () => {
     }
 
     for (const dns of dnsNames) {
-      runPlaybook('stop_morio.yml', ansibleDir, dns);
+      runPlaybook('stop_moriod.yml', ansibleDir, dns);
```

Comment:
This goes a bit against the way things are done in Ansible.
I'm sure it works, but it is going to be super confusing for people who are used to Ansible.

I think what would be a better way to go about this is to have our very first step (where we deploy the test infratstructure create an ansible inventory file.

Then, we can just run the various ansible playbooks and specify the targets. Ansible will handle all of the parallel processing for us.

Here is an example Ansible inventory file. I am using JSON here because that way we can also load this easily in NodeJS

```json
{
  "local":{
    "hosts":{
      "localhost":{
        
      }
    }
  },
  "all":{
    "children":{
      "pr98":{
        "hosts":{
          "abc.test.morio.it":{
            "example_variable":"example_value"
          }
        },
        "pr99":{
          "hosts":{
            "def.test.morio.it":{
              "example_variable":"example_value"
            },
            "ghi.test.morio.it":{
              "example_variable":"example_value"
            },
            "jkl.test.morio.it":{
              "example_variable":"example_value"
            }
          }
        }
      }
    }
  }
}
```

This allows us to combine ansible's `-i` (or `--inventory-file`) flag and its `-l` flag (or `--limit`) to limit the playbook run to those hosts we want. Eg:

```
ansible-playbook -i ./ansible/inventory.json -l pr99 ./ansible/some-playbook.yml
```

Or for improved legibility:
```
ansible-playbook --inventory-file ./ansible/inventory.json --limit pr99 ./ansible/some-playbook.yml
```

Reply:
Also, as I am writing this, a thought occurred to me: Perhaps using Terraform to deploy the initial infrastructure is overkill 🤔 

Terraform really shines when keeping infrastructure in sync with your desired configuration over time. But for that, we would need to store the terraform state file somewhere, something which we are currently not doing.

However, we do not really need consistency over time. We can just create and destroy, using labels/tags to ensure everything gets cleaned up.

So perhaps it makes more sense to deploy the infrastructure with ansible, using the aws module: https://docs.ansible.com/ansible/latest/collections/amazon/aws/index.html

This will mean one less thing (terraform) to manage and have to understand before one can contributor meaningfully to the project.

Curious to hear your thoughts on this.

Reply:
I've changed all deploy, setup, destroy, reset scenario as a direct ansible playbook.